packages/sanity/src/core/studio/components/navbar/resources/ResourcesMenuItems.tsx

```diff
@@ -0,0 +1,123 @@
+import {Box, Card, Flex, Label, MenuDivider, MenuItem, Spinner, Text} from '@sanity/ui'
+import React from 'react'
+import {SANITY_VERSION} from '../../../../version'
+import {ResourcesResponse, Section} from './helper-functions/types'
+
+interface ResourcesMenuItemProps {
+  error: Error | null
+  isLoading: boolean
+  value: ResourcesResponse | undefined
+}
+
+export function ResourcesMenuItems({error, isLoading, value}: ResourcesMenuItemProps) {
+  const sections = value?.resources?.sectionArray
+  const latestStudioVersion = value?.latestVersion
+
```

Comment:
I would consider doing early returns instead of inline conditionals - it makes it easier to read the code, since you already know the different cases are accounted for. Not a blocker, but for future reference (psuedo-code):

```
if (isLoading) {
  return <LoadingUI />
}

return <NonLoadingUiStuffHere />
```